board/recalbox/fsoverlay/etc/init.d/S12populateshare

```diff
@@ -33,7 +33,7 @@ for FILE in music bios extractions kodi saves screenshots \
             system/.config/lirc/lircd.conf \
             system/.emulationstation/{es_input.cfg,es_settings.cfg} \
             system/{.hatari,.kodi} \
-            system/configs/{fba,moonlight,mupen64,ppsspp,reicast,retroarch,shadersets,xarcade2jstick} \
+            system/configs/{fba,moonlight,mupen64,ppsspp,reicast,retroarch,shadersets,xarcade2jstick,dosbox} \
```

Comment:
Juste a note : we should really find another way for `~/configs` creation/upgrade. Would like feedback on https://github.com/recalbox/recalbox-os/issues/1043
And so far `/recalbox/share_init/system/configs/dosbox` doesn't exist, but a `.dosbox` in system. Should settle these down